plugins/Jobs.cpp

```diff
@@ -266,20 +350,25 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
             }
         }
 
-
         // If no priority set, set it
         int64_t priority = request.isSet("priority") ? request.calc("priority") : JOBS_DEFAULT_PRIORITY;
 
         // Validate that the parentJobID exists and is in the right state if one was passed.
         int64_t parentJobID = request.calc64("parentJobID");
         if (parentJobID) {
-            auto parentState = db.read("SELECT state FROM jobs WHERE jobID=" + SQ(parentJobID) + ";");
-            if (parentState.empty()) {
+            SQResult result;
+            if (!db.read("SELECT state, retryAfter FROM jobs WHERE jobID=" + SQ(parentJobID) + ";", result)) {
+                throw "502 Select failed";
+            }
+            if (result.empty()) {
                 throw "404 parentJobID does not exist";
-            } else if (!SIEquals(parentState, "RUNNING") && !SIEquals(parentState, "PAUSED")) {
-                SWARN("Trying to create child job with parent jobID#" << parentJobID << ", but parent isn't RUNNING or PAUSED (" << parentState << ")");
+            } else if (!SIEquals(result[0][0], "RUNNING") && !SIEquals(result[0][0], "PAUSED")) {
+                SWARN("Trying to create child job with parent jobID#" << parentJobID << ", but parent isn't RUNNING or PAUSED (" << result[0][0] << ")");
                 throw "405 Can only create child job when parent is RUNNING or PAUSED";
             }
+            else if (result[0][1] != "") {
+                SALERT("Auto-retrying parents shouldn't have children, parentJobID:" << parentJobID );
```

Comment:
I'm throwing 402 in peekCommand and I added this in processCommand just as a sanity check
```cpp
+            if (!result.empty() && result[0][0] != "") {
+                throw "402 Auto-retrying parents cannot have children";
+            }
```

Should I throw 405 instead of 402 in peek?